mobile/test/common/integration/test_server.h

```diff
@@ -10,6 +10,13 @@
 #include "test/mocks/server/transport_socket_factory_context.h"
 
 namespace Envoy {
+
+enum TestServerType {
+  HTTP1_HTTP,
+  HTTP2_HTTPS,
```

Comment:
@RyanTheOptimist would be more suited to give naming feedback on HTTP protocol naming, but this seems a little confusing to me at first glance, I would find this more clear: `{ HTTP1, HTTP2_WITH_TLS, HTTP3 }` (I don't think you can have H3 w/out TLS, right?)

Reply:
I should say `{ HTTP1_WITHOUT_TLS, HTTP2_WITH_TLS, HTTP3 }`

Reply:
Those names sound reasonable to me.

Reply:
Changed